{{cookiecutter.project_slug}}/frontend/src/main.ts

```diff
@@ -1,19 +1,15 @@
-import '@babel/polyfill';
-// Import Component hooks before component definitions
-import './component-hooks';
-import Vue from 'vue';
-import './plugins/vuetify';
-import './plugins/vee-validate';
-import App from './App.vue';
-import router from './router';
-import store from '@/store';
-import './registerServiceWorker';
-import 'vuetify/dist/vuetify.min.css';
+import Vue from "vue";
```

Comment:
This import is necessary to get vue router redirects working again (so that  `beforeRouteEnter` gets called)
```suggestion
import "./component-hooks";
import Vue from "vue";
```